client/i18n/locales/english/translations.json

```diff
@@ -555,7 +555,9 @@
     "info": "Intro Information",
     "spacer": "Spacer",
     "toggle": "Toggle Checkmark",
-    "magnifier": "Submit search terms"
+    "magnifier": "Submit search terms",
+    "bear-progress": "A cute teddy bear holding a donation jar",
+    "bear-block-completion": "A cute teddy bear holding a gold cup"
```

Comment:
About the purpose of the bear, if we have it for decoration, we should remove the `sr-only` text.

Otherwise, and I am misunderstanding it, we can keep the text.

```suggestion
```

Reply:
Thanks for the review. The sr-only has been taken out as requested.